### DISCUSSION TITLE
Add to edge webstore

### DISCUSSION THREAD

**Discussion by mx-raza** [arimgibson/Deadname-Remover#568] [Ideas]
I love your Deadname remover extension and have been using it for a while.
I want to use it on my company device but it needs an approval from the company for it and in order to do so it needs to be on the Microsoft Edge Add-ons webstore. It would also be good for it to be added there to be used in the android edge.
Would it be possible for you to add it to the edge webstore?
Thanks!

**Comment by arimgibson** [comment]
@mx-raza love it, yes! I will definitely look into this -- I have finally found enough time to actually put in a large amount of effort into this project, so there's a new version that'll come out in the next week or two. Once I get that live on the current platforms, I will get everything setup to publish to the Edge add-ons store as well.

Wasn't familiar with Android Edge extensions but that's a really great feature -- I don't see any reason why the extension wouldn't work on Android as well :blush:
